README.md

```diff
@@ -13,10 +13,15 @@
 - In case of setting up synchronization, no whitespace is allowed in the path to the project directory.
 
 # Setup
-### For Ubuntu,
+### For Unix-like OS
```

Comment:
If the section only concerns Fedora Linux and Ubuntu Linux, use the word _Linux_ in the title. _Unix-like_ can include macOS, which is similar but different in many ways.

Reply:
This update is based on the comment in #664 by Yonguk. And I agree with you, we need to add macOS as another case. But I forget the setup process on macOS and it is quite hard to get a clean macOS now.

Reply:
I get the point. in that case, we should point out what the 1st step is: install python and pq packages, instead of bluntly saying _Do + some command_. So for example, change this step to "Install Python 3", and "Install LIBPQ" (whatever libpq is).

Meanwhile, let's add a placeholder for macOS, saying that the detailed step is coming soon, and please refer to the Linux installation guide for now. So the user won't be surprised by the Unix-like section, which does not explicitly say anything about macOS, but hoping the user to figure out which of the 4 steps work on their computer, and which does not.